android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsStatus.kt

```diff
@@ -0,0 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+/**
+ * Value type that represents the status of a translation within a [SessionState].
+ *
+ * @property isTranslated The page is currently translated.
+ * @property isTranslateProcessing The page is currently attempting a translation.
+ * @property isTranslateProcessing The page is currently attempting a restoration.
```

Comment:
```suggestion
 * @property isRestoreProcessing The page is currently attempting a restoration.
```

Reply:
Thanks, good eye! Kinda surprised the linter didn't catch that. I think the GeckoView lints would have 🤔.

Reply:
I am working on it with https://bugzilla.mozilla.org/show_bug.cgi?id=1848527. The changes needed to made for AC is big to say the least so I am still slowly adding comments when I have some motivation to do that.